src/perl/vdotest/VDOTest/VDOClobber.pm

```diff
@@ -0,0 +1,121 @@
+##
+# Test that creating a vdo doesn't clobber a file system or another VDO volume
+#
+# $Id: $
+##
+package VDOTest::VDOClobber;
+
+use strict;
+use warnings FATAL => qw(all);
+use English qw(-no_match_vars);
+use Log::Log4perl;
+
+use Permabit::Assertions qw(
+  assertEvalErrorMatches
+  assertNumArgs
+  assertRegexpMatches
+);
+use Permabit::CommandString::VDOFormat;
+use Permabit::Constants;
+use Permabit::PlatformUtils qw(getDistroInfo);
+
```

Comment:
The VDOFormat ComandString doesn't sem to get used in theses tests.
I'm not usre what you're using from Constants, that might also not be needed?
getDistroInfo() is used, but as I comment below I don't think it's necessary, either.

Reply:
I've removed VDOFormat, Constants and getDistroInfo since I removed the btrfs as well

Reply:
Constants is still included but it might be needed for something, sure.